addons/website_profile/views/website_profile.xml

```diff
@@ -531,24 +531,26 @@
                     <t t-call="website_profile.top3_user_card"></t>
                 </div>
             </div>
-            <table class="table table-sm" t-if='users'>
-                <t t-foreach="users" t-as="user">
-                    <tr t-attf-onclick="location.href='/profile/user/#{user['id']}';" t-attf-class="o_wprofile_pointer bg-white #{user['id'] == user_id.id and 'o_wprofile_border_focus'}">
-                        <t t-call="website_profile.all_user_card"/>
-                    </tr>
-                </t>
-                <t t-if="my_user">
-                    <!-- keep same table to avoid missaligment -->
-                    <tr>
-                        <td colspan="7"></td>
-                    </tr>
-                    <t t-set='user' t-value='my_user'/>
-                    <tr t-attf-onclick="location.href='/profile/user/#{user['id']}';" t-attf-class="o_wprofile_pointer bg-white o_wprofile_border_focus">
-                        <t t-call="website_profile.all_user_card">
-                        </t>
-                    </tr>
-                </t>
-            </table>
+            <div class="table-responsive" t-if="users">
+                <table class="table table-sm">
```

Comment:
Let's not change this in stable even though it's redundant, just in case someone used it.

```suggestion
                <table class="table table-sm" t-if='users'>
```